_data/sites.json

```diff
@@ -3459,8 +3469,8 @@
         "name": "Dreamstime",
         "url": "https://www.dreamstime.com/account/edit-profile",
         "difficulty": "easy",
-        "notes": "Click on the \"Delete account\" link towards the bottom of the page and then confirm your action in the opened pop-up.",
-        "notes_tr": "Sayfanın alt kısmına doğru gidin ve \"Hesabı Sil\" bağlantısını tıklayın ve ardından açılan pencerede işleminizi onaylayın.",
+        "notes": "Click on the \"Delete account\" link towards the bottom of the page and then confirm your action in the opened pop-up. Accounts can be deleted only if you have no activity registered (uploads, downloads, comments,blogs etc).",
+        "notes_tr": "Sayfanın alt kısmına doğru gidin ve \"Hesabı Sil\" bağlantısını tıklayın ve ardından açılan pencerede işleminizi onaylayın. Hesaplar yalnızca kayıtlı hiçbir etkinliğiniz yoksa (yüklemeler, indirmeler, yorumlar, bloglar vb.) silinebilir.",
```

Comment:
Do you speak Turkish? If not we recommend entries of languages you do not speak to be deleted instead.

Reply:
@tupaschoal I don't. I used deepl to append the additional information.

Reply:
Then can you please remove that line?